projects/core/interfaces/notification-options.ts

```diff
@@ -0,0 +1,20 @@
+import {TuiContextWithImplicit} from '@taiga-ui/cdk';
+import {TuiNotification} from '@taiga-ui/core/enums';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+export type TuiNotificationAutoClose =
```

Comment:
There's a folder `types` for types :) @vladimirpotekhin @nsbarsukov @Smirnov906 I wonder if it makes sense to keep this distinction 🤔

Reply:
I think it makes no sense

Reply:
@waterplea It all depends on the style guides of the project )) I don't know yet what the rules are